### ISSUE TITLE
Experimental models like broken

### ISSUE THREAD

**Issue by erickrf** [Helsinki-NLP/Opus-MT#56]
The link mentioned in the Readme to experimental models seems broken or with restricted access: https://github.com/Helsinki-NLP/Opus-MT-train/tree/master/work-spm/models

**Comment by jorgtied** [comment]
Good catch. I didn't update that list for a long time and decided to remove those sub-directories to make the repository a bit more easy to handle. Is there a specific model that you would need?

**Comment by erickrf** [comment]
I was wondering if there was a German-Slovak model.

**Comment by jorgtied** [comment]
I can check but I could also start a job to train one. Maybe a German --> Czech + Slovak would be a good choice?

**Comment by erickrf** [comment]
Thanks, that would be great! I'm not an expert on these languages but I believe this combination makes sense.

**Comment by jorgtied** [comment]
@erickrf: I have uploaded a model for de-sk now: https://object.pouta.csc.fi/OPUS-MT-models/de-sk/opus_transformer-align_2022-02-19.zip. The multilingual model with Czech is still running. More info about the de-sk model is here: https://object.pouta.csc.fi/OPUS-MT-models/de-sk/opus_transformer-align_2022-02-19.yml

**Comment by erickrf** [comment]
Thanks! How can I create a huggingface transformers configuration from the checkpoint? Is it by the way much less efficient than using Marian directly?
